app/NewOrder.php

```diff
@@ -40,7 +40,10 @@ public static function rules()
             'discount' => 'sometimes|array',
             'discount.*' => 'sometimes|numeric|exists:discounts,id',
             'product_price' => ['required', new Money],
-            'custom_date' => 'integer|min:1|max:31|required_if:repayment_cycle_id,' . $id
+            'custom_date' => 'integer|min:1|max:31|required_if:repayment_cycle_id,' . $id,
+            'down_payment_rate_id' => 'required|exists:down_payment_rates,id',
+            'order_type_id' => 'required|exists:order_types,id',
+            'payment_gateway_id' => 'required|exists:payment_gateways,id',
```

Comment:
do we want to require it? what about Altara credit that doesn't have to do with Remitta or Paystack?